packages/cli/lib/dependencies.json

```diff
@@ -19,13 +19,13 @@
   "debug": "^3.1.0",
   "nyc": "^11.4.1",
   "strong-docs": "^1.7.1",
-  "@loopback/authentication": "^0.2.0",
-  "@loopback/cli": "^0.2.0",
-  "@loopback/metadata": "^0.2.0",
-  "@loopback/openapi-spec-builder": "^0.2.0",
-  "@loopback/openapi-spec": "^0.2.0",
-  "@loopback/openapi-v3-types": "^0.2.0",
-  "@loopback/openapi-v3": "^0.2.0",
-  "@loopback/repository-json-schema": "^0.2.0",
-  "@loopback/repository": "^0.2.0"
+  "@loopback/authentication": "^0.3.0",
+  "@loopback/cli": "^0.3.0",
+  "@loopback/metadata": "^0.2.1",
+  "@loopback/openapi-spec-builder": "^0.3.0",
+  "@loopback/openapi-spec": "^0.3.0",
+  "@loopback/openapi-v3-types": "^0.2.1",
+  "@loopback/repository-json-schema": "^0.2.1",
+  "@loopback/repository": "^0.2.1",
+  "@loopback/openapi-v2": "^0.3.0"
```

Comment:
Hmm, do we still want it to be a dependency? The version is good.

Reply:
After your PR to drop v2 support, it will be gone. The extra dep won't harm.